server/utils/connection.js

```diff
@@ -0,0 +1,30 @@
+const { sequelize } = require("../models");
+const mongoose = require("mongoose");
+
+const startServer = async (app, port) => {
+  try {
+    // Wait for PostgreSQL to be ready
+    await sequelize.authenticate();
+    console.log("PostgreSQL connection has been established successfully.");
+
+    // Sync Sequelize models
+    await sequelize.sync();
+    console.log("PostgreSQL models synced successfully.");
+
+    // Wait for MongoDB to be ready
+    await mongoose.connect(process.env.MONGODB_URI, {
```

Comment:
How do we generally ensure that all required envs are set in process.env? any ways to make system missing-env tolerant?

Reply:
Currently it cannot be handled but we could create a common utility to handle this. But if the env is not defined then it will fail and will alert the dev right?